### PR TITLE
Add MIME type

### PR DESCRIPTION
**Pull request by Partmedia**
Add model/slvs MIME type, which allows .slvs files to be opened from FreeDesktop-compliant graphical environments.

### CODE REVIEW

**Comment by whitequark** [comment]
Thank you for your contribution, and welcome. The SolveSpace project has a Contributor License Agreement; in order to merge your pull request, please sign it at https://cla-assistant.io/solvespace/solvespace. It will take only a small amount of your time.

**Comment by Partmedia** [comment]
I've agreed to the CLA.

File: res/freedesktop/solvespace.desktop.in [hunk 1]
```diff
@@ -3,6 +3,7 @@ Version=1.0
 Name=SolveSpace
 Comment=A parametric 2d/3d CAD
 Exec=${CMAKE_INSTALL_FULL_BINDIR}/solvespace
+MimeType=model/slvs
```
Comment (whitequark) [inline res/freedesktop/solvespace.desktop.in]: Shouldn't this be something like `application/x-solvespace`? I think all MIME types not explicitly assigned by IANA should be in the `x-` namespace.

Comment (Partmedia) [reply res/freedesktop/solvespace.desktop.in]: You're right. I'll make the change.

**Comment by Partmedia** [comment]
Fixed. If it's satisfactory I can squash and rebase.

**Comment by whitequark** [comment]
Thank you again!

**Pull request merged**